components/monitoring/prometheus/base/monitoringstack/monitoringstack.yaml

```diff
@@ -99,3 +99,45 @@ spec:
         configMap: # automatically created by serving-ca operator
           key: service-ca.crt
           name: openshift-service-ca.crt
+---
+# Create ServiceMonitor for user workload prometheus Federation
+apiVersion: monitoring.rhobs/v1
+kind: ServiceMonitor
+metadata:
+  name: appstudio-federate-uwm-smon
+  namespace: appstudio-monitoring
+  labels:
+    kubernetes.io/part-of: appstudio-federate-ms
+    monitoring.rhobs/stack: appstudio-federate-ms
+spec:
+  selector: # use the prometheus service to create a "dummy" target.
+    matchLabels:
+      app.kubernetes.io/managed-by: observability-operator
+      app.kubernetes.io/name: appstudio-federate-ms-prometheus
+  endpoints:
+  - params:
+      'match[]': # scrape only required metrics from UWM prometheus
+        - '{__name__=~".*"}' # all the metrics from UWM prometheus
```

Comment:
is this supposed to change in the future? since you noted on 119 this is used to scrape only required metrics, but here we seem to scrape them all.

Reply:
For now we want all the metrics from UWM, and we left this placeholder in case we would like to limit it in the future